lib/rules/string-quotes/index.js

```diff
@@ -187,7 +188,7 @@ const rule = (primary, secondaryOptions, context) => {
 				return;
 			}
 
-			if (node.type === 'atrule' && node.name === 'charset') {
+			if (isAtRule(node) && node.name === 'charset') {
 				// allow @charset rules to have double quotes, in spite of the configuration
 				// TODO: @charset should always use double-quotes, see https://github.com/stylelint/stylelint/issues/2788
```

Comment:
seems like you can remove the "TODO: " prefix

Reply:
PR welcome!